problem_builder/public/css/dashboard.css

```diff
@@ -24,7 +32,7 @@
     min-width: 4em;
     text-align: right;
     padding-right: 5px;
-    border-right: 0.6em solid transparent;
+    border-right: 1.875em solid transparent;
```

Comment:
Why 1.875? If the goal is to make them square, shouldn't it be 2? 2 certainly looks more square when I try it.